exporters/prometheus/testdata/histogram.txt

```diff
@@ -9,7 +9,12 @@ baz_bucket{A="B",C="D",le="75"} 0
 baz_bucket{A="B",C="D",le="100"} 0
 baz_bucket{A="B",C="D",le="250"} 2
 baz_bucket{A="B",C="D",le="500"} 0
+baz_bucket{A="B",C="D",le="750"} 0
 baz_bucket{A="B",C="D",le="1000"} 0
+baz_bucket{A="B",C="D",le="2500"} 0
+baz_bucket{A="B",C="D",le="5000"} 0
+baz_bucket{A="B",C="D",le="7500"} 0
+baz_bucket{A="B",C="D",le="10000"} 0
```

Comment:
Maybe we should setup specific bounds here to avoid having to update tests whenever those change (even though they shouldn't change very often either).

Reply:
Also, now that our bounds match prometheus' default bounds, it would ensure our configuration is actually taking effect.

Reply:
I've added a view to use custom buckets for this histogram.